src/utils/docxNodes.ts

```
@@ -0,0 +1,91 @@
+import { defaultNodes } from 'prosemirror-docx'
+import type { DocxSerializerState } from 'prosemirror-docx'
+import type { Node } from 'prosemirror-model'
+
+interface DocxSerializerStateExtended extends DocxSerializerState {
+  write: (content: any) => void;
+}
+
+interface NodeHandler {
+  (state: DocxSerializerStateExtended, node: Node, ...args: any[]): void
```

Biome (1.9.4)

[error] Use a function type instead of a call signature.

Types containing only a call signature can be shortened to a function type.
Safe fix: Alias a function type instead of using an interface with a call signature.


(lint/style/useShorthandFunctionType)